picard/ui/options/dialog.py

```diff
@@ -150,7 +149,7 @@ def __init__(self, default_page=None, parent=None):
         # work-around to set optimal option pane width
         self.ui.pages_tree.expandAll()
         max_page_name = self.ui.pages_tree.sizeHintForColumn(0) + 2*self.ui.pages_tree.frameWidth()
-        self.ui.splitter.setSizes([max_page_name,
+        self.ui.dialog_splitter.setSizes([max_page_name,
```

Comment:
The indentation of the next line needs to be updated (not sure it needs to be wrapped though).

Unrelated to this PR, but what about:

```python
left = self.ui.pages_tree.sizeHintForColumn(0) + 2*self.ui.pages_tree.frameWidth()
right = self.geometry().width() - left
self.ui.dialog_splitter.setSizes([left, right])
```

And stick with such notation for splitters, (left/right, top/bottom).

Reply:
That makes it a lot easier to understand.  I like it.